src/support/timefuncs.c

```diff
@@ -37,7 +38,7 @@ JL_DLLEXPORT double jl_clock_now(void)
 {
     struct jl_timeval now;
     jl_gettimeofday(&now);
-    return now.sec + now.usec * 1e-6;
+    return now.sec + now.nsec * 1e-9;
 }
 
 void sleep_ms(int ms)
```

Comment:
Should the `select` in this function really be used here? I think using a different API for sleeping like `nanosleep` would be more appropriate, as it can also consume seconds & nanoseconds like this PR enables via `time()`.